runtime/server/api.go

```diff
@@ -0,0 +1,109 @@
+package server
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/rilldata/rill/runtime"
+	"github.com/rilldata/rill/runtime/pkg/observability"
+	"github.com/rilldata/rill/runtime/server/auth"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+// nolint
```

Comment:
Why `nolint`?

Reply:
It had some commented code around how resolver is expected to get called. It is temporary. Can be removed once resolver PR is integrated.